### ISSUE TITLE
missing default pages and navbar

### ISSUE THREAD

**Issue by litlfred** [WorldHealthOrganization/smart-ig-empty#9]
This is missing the L1-L4 navbar and empty pages that are (almost) in the smart-ig-starter kit.

**Comment by litlfred** [comment]
in individual files would also be good to substitute IG name with jekyll magic where possible.  for example in testing.md
there are hard coded references like "https://raw.githubusercontent.com/WorldHealthOrganization/smart-immunizations-measles/main/testing/docker/logic/docker-compose.yml

**Comment by costateixeira** [comment]
yes, that is the plan. the .env files should contain all of that, and I
want the .env file to be created by the publishing template.
